pkg/agent/agent.go

```diff
@@ -295,7 +296,7 @@ func (i *Initializer) initInterfaceStore() error {
 					var err error
 					intf, err = externalnode.ParseHostInterfaceConfig(i.ovsBridgeClient, port, ovsPort)
 					if err != nil {
-						return err
+						return fmt.Errorf("failed to get interfaceConfig by port %s: %s", port.Name, err.Error())
```

Comment:
return fmt.Errorf("failed to get interfaceConfig by port %s: %v", port.Name, err)

Reply:
Done